backend/package.json

```diff
@@ -4,15 +4,19 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "test": "jest --colors",
+    "test": "jest --colors --runInBand",
```

Comment:
this makes every test run sequentially, why is it needed? By running everything parallel you increase the chance of finding side effects, dont you?